lib/chef/knife/ec2_server_create.rb

```diff
@@ -453,6 +453,10 @@ class Ec2ServerCreate < Knife
         :boolean => true,
         :default => false
 
+      option :instance_initiated_shutdown_behavior,
+        :long => "--instance-initiated-shutdown-behavior",
+        :descript => "Indicates whether an instance stops or terminates when you initiate shutdown from the instance. Possible values are 'stop' and 'terminate', default is 'stop'."
```

Comment:
Please add `:description =>` instead of `:descript =>` as per naming convention used by all above examples.